react/fields/Checkbox/Checkbox.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-block;
+}
+
+.input {
+  opacity: 0;
+  position: absolute;
+}
+
+.label {
+  .touchableText(@standard-type-scale);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+}
+
+.checkBox {
+  width: @grid-row-height * 3;
+  height: @grid-row-height * 3;
```

Comment:
should the height/width take into account the border size so that the component stays on grid?